package.json

```diff
@@ -40,6 +40,7 @@
   "devDependencies": {
     "@babel/core": "7.16.0",
     "@babel/parser": "7.16.0",
-    "lerna": "^4.0.0"
+    "lerna": "^4.0.0",
+    "@mongodb-js/bump-monorepo-packages": "file:./packages/bump-monorepo-packages"
```

Comment:
Having it as a dev dep like that still seems a bit off to me, especially because in theory for workspaces it should be a version here and I'm worried that having a file path here might mess up the npm dependencies tree. In compass monorepo even without having this compass-scripts package that is part of the monorepo as a dependency in the root I am able to run them both with npx and with npm scripts:

![image](https://user-images.githubusercontent.com/5036933/180474569-866ca3f6-a5f9-4ba8-a5f1-28f2a7d72dab.png)

![image](https://user-images.githubusercontent.com/5036933/180474602-455e36d6-5b29-4f0f-88f4-913155e6a74c.png)

Reply:
interesting, i couldn't make it work, have to try again